sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_recents_v1/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+friendly_name: Experiment Enrollment Aggregates Recents
+description: >
+  Experiment enrollment aggregates data that has not been processed by
+  the hourly job.
+labels:
+  incremental: false
+owners:
+- ascholtz@mozilla.com
+scheduling:
+  dag_name: bqetl_experiments_live
+  task_name: experiment_enrollment_aggregates_recents
+  parameters: ["submission_timestamp:TIMESTAMP:{{ts}}"]
+  referenced_tables: [
+    ["moz-fx-data-shared-prod", "telemetry_derived", "events_live"]
```

Comment:
`events_live` is a virtual view, so shouldn't appear in `referenced_tables`. We should replace these references with the underlying tables that the view hits.